auto_round/export/export_to_gguf/quant.py

```diff
@@ -165,216 +167,181 @@ def q8_0_quant_block(blocks: np.array, scale=None, zp=None) -> np.ndarray:
 
     return np.concatenate([d, qs], axis=1)
 
+def make_qkx2_quants(data, bits, rmin=-1, rdelta=0.1, nstep=20, use_mad=False):
+    assert bits in [2, 4], f"bits={bits} is not supported"
+    nmax = bits ** 2 -1
+    # data shape (nb, 8, 32) for Q4_K, (nb, 16, 16) for Q2_K
+    if len(data.shape) == 2:
```

Comment:
assert bits==2 or 4

Reply:
For other bits that use asymmetric quantization, we should add support for them later.